bin/bundle-accp-jdk8.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+set -e
+set -x
+
+# Use this script for JDKs 8. Please note that this script does not modify the security file,
+# so one still needs to programmatically install ACCP.
+#
+# This script requires JAVA_HOME set properly.
+#
+# usage: ./bundle-script 2.3.3 linux-x86_64
+
+ACCP_VERSION=$1 # for example 2.3.3
+ACCP_PLATFORM=$2 # for example linux-x86_64 or osx-aarch64
+FIPS_EXTENTION=${3:""} # for FIPS, pass -FIPS
+BUILD_DIR="$(pwd)/build"
+TMP_DIR=$BUILD_DIR/private/tmp
+ACCP_PATH=$TMP_DIR/accp
+JDK_PATH=$JAVA_HOME
+JAVA_BIN=$JDK_PATH/bin
+
+function download_accp {
+    wget -O $ACCP_PATH/accp.jar https://repo1.maven.org/maven2/software/amazon/cryptools/AmazonCorrettoCryptoProvider${FIPS_EXTENTION}/${ACCP_VERSION}/AmazonCorrettoCryptoProvider${FIPS_EXTENTION}-${ACCP_VERSION}-${ACCP_PLATFORM}.jar
```

Comment:
Here and everywhere where a path is constructed from variables like `$ACCP_PATH`, I would add double quotes.

Concretely:

```
wget -O "$ACCP_PATH/accp.jar" https://repo1.maven.org/maven2/software/amazon/cryptools/AmazonCorrettoCryptoProvider${FIPS_EXTENTION}/${ACCP_VERSION}/AmazonCorrettoCryptoProvider${FIPS_EXTENTION}-${ACCP_VERSION}-${ACCP_PLATFORM}.jar
```

or below

```
./gradlew -PuseBundledAccp -Dorg.gradle.java.home="$JDK_PATH" lib:test
```

Without double quotes, the script will fail when the current path contains spaces.

In the assignments `TMP_DIR=$BUILD_DIR/private/tmp`, quotes are not mandatory, but I always prefer to put them there too, to be on the safe side.

See https://unix.stackexchange.com/a/68748